pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/codestyle/xml/ArrayInitializationVerboseness.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<test-data
+  xmlns="http://pmd.sourceforge.net/rule-tests"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://pmd.sourceforge.net/rule-tests http://pmd.sourceforge.net/rule-tests_1_0_0.xsd">
+  <test-code>
+    <description><![CDATA[
+simple failure case
+     ]]></description>
+    <expected-problems>1</expected-problems>
+    <code><![CDATA[
+public class Foo {
+ void foo() {
+  int[] x = new int[] {1,2,3};
+ }
+}
+     ]]></code>
+  </test-code>
+
+  <test-code>
+    <description><![CDATA[
+case with two initializers
+     ]]></description>
+    <expected-problems>2</expected-problems>
+    <code><![CDATA[
+public class Foo {
+ void foo() {
+   int ar[] = new int[] { 1,2,3}, foo[] = new int[] { 4, 5, 6 };
```

Comment:
You also need negative test cases like 
```java
int[] a = { 1 };
a = new int[] { 2 }; // can't be replaced 
```
 
```java
int[] a = new int[3]; // no violation
```

Also the initializer syntax is available for field initializers and it would be nice to cover it in tests as well.